TriggerHandler.lua

```diff
@@ -408,7 +408,9 @@ end
 
 function TriggerHandler:needsFuel()
 	local dieselIndex = self.vehicle:getConsumerFillUnitIndex(FillType.DIESEL)
-	local currentFuelPercentage = self.vehicle:getFillUnitFillLevelPercentage(dieselIndex) * 100
+	local electricIndex = self.vehicle:getConsumerFillUnitIndex(FillType.ELECTRICCHARGE)
```

Comment:
Can we consolidate this fuel checking into a single function like ADriverUtil.getCurrentFuelPercentage() and use that from here and from AIDriver?

Reply:
Will have a look at this one, could probably consolidate this one with AIDriver:checkFuel().